pyrobosim/test/utils/test_pose_utils.py

```diff
@@ -40,13 +41,20 @@ def test_pose_from_position():
     assert pose.q == pytest.approx([1.0, 0.0, 0.0, 0.0])
 
 
-def test_pose_from_euler_angles():
-    """Test creating a pose using Euler angles."""
-    pose = Pose(roll=np.pi / 2, pitch=0.0, yaw=-np.pi / 2)
+@pytest.mark.parametrize(
+    "angle_units, roll, pitch, yaw, expected_euler",
+    [
+        ("radians", np.pi / 2, 0.0, -np.pi / 2, [np.pi / 2, 0.0, -np.pi / 2]),
+        ("degrees", 90.0, 0.0, -90.0, [np.pi / 2, 0.0, -np.pi / 2]),
+    ],
+)
```

Comment:
nice! Although seems like the `expected_euler` is the same in both cases, so probably don't need to include that in these args.